server/authmanager.go

```diff
@@ -227,3 +227,13 @@ func AuthorizationRoundTripper(config *Config, next http.RoundTripper) (http.Rou
 		return next.RoundTrip(r)
 	}), nil
 }
+
+// NoAuthPUTRoundTripper strips the authorization header for any PUT request
```

Comment:
The what is clear in the naming and code, it would be more useful for the comment to explain why.

Reply:
Hope it self-explains better now.